packages/std/src/addresses.rs

```diff
@@ -23,8 +24,18 @@ use crate::binary::Binary;
 /// This type is immutable. If you really need to mutate it (Really? Are you sure?), create
 /// a mutable copy using `let mut mutable = Addr::to_string()` and operate on that `String`
 /// instance.
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, Hash, JsonSchema)]
-pub struct Addr(String);
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash, JsonSchema)]
+pub struct Addr(#[schemars(with = "String")] AddrInner);
```

Comment:
The two level nesting here is for schemars output?

Reply:
Eaxtly. If we find a better way to ensure it gets a "string" schema, we can simplify at any time because all the internals are private.